doc/source/theory/FlavorSpace.rst

```diff
@@ -75,31 +76,126 @@ The mapping between the Evolution Basis and the +/- Basis is given by
+  T_1^d &=d^+ - s^+ \\
+  T_2^d &=d^+ + s^+ - 2b^+ \\
+  V_1^u &=u^- - c^- \\
+  V_2^u &=u^- + c^- - 2t^- \\
+  V_1^d &=d^- - s^- \\
+  V_2^d &=d^- + s^- - 2b^- \\
+  \mathcal F \sim  \mathcal F_{QED\otimes{}QCD} &= \span(\gamma, g, \Sigma, \Sigma_0, V, V_0, T_1^u, T_2^u, T_1^d, T_2^d, V_1^u, V_2^u, V_1^d, V_2^d)
+
+
+- :math:`T_1` and :math:`T_2` are just arbitrary names and do not follow any group representation like it was for the :ref:`theory/FlavorSpace:QCD Evolution Basis`
+- this basis is *not* normalized with respect to the canonical Flavor Basis
+- The singlet :math:`\Sigma` is just the QCD singlet
+- The valence :math:`V` is just the QCD valence
+
+
+Intrinsic QED :math:`\otimes` QCD Evolution Basis
+-------------------------------------------------
+
+Again, we need the generalization to the presence of intrinsic (static) distributions.
+As QED can distinguish between up-like and down-like flavors the situation is again slightly
+more involved.
+
+For :math:`n_f=3` light flavors we find:
+
+.. math ::
+  \Sigma_{(3)} &= u^+ + d^+ + s^+\\
+  T_{0,(3)} &= 2u^+ - d^+ - s^+ \\
+  V_{(3)} &= u^- + d^- + s^-\\
+  V_{0,(3)} &= 2u^- - d^- - s^-\\
+  T_1^d &=d^+ - s^+ \\
+  V_1^d &=d^- - s^- \\
+  \mathcal F \sim  \mathcal F_{intrinsic_unified_evol,3} &= \span(\gamma, g, \Sigma_{(3)}, V_{(3)}, V_{0,(3)}, V_1^d, T_{0,(3)}, T_1^d, c^+, c^-, b^+, b^-, t^+, t^-)
+
+For :math:`n_f=4` light flavors we find:
+
+.. math ::
+  \Sigma_{(4)} &= u^+ + d^+ + s^+ + c^+\\
+  T_{0,(4)} &= u^+ + c^+ - d^+ - s^+\\
+  V_{(4)} &= u^- + d^- + s^- + c^-\\
+  V_{0,(4)} &= u^- + c^- - d^- - s^-\\
+  T_1^u &=u^+ - c^+ \\
+  T_1^d &=d^+ - s^+ \\
+  V_1^u &=u^- - c^- \\
+  V_1^d &=d^- - s^- \\
+  \mathcal F \sim  \mathcal F_{intrinsic_unified_evol,4} &= \span(\gamma, g, \Sigma_{(4)}, V_{(4)}, V_{0,(4)}, V_1^d, T_{0,(4)}, T_1^d, V_1^u, T_1^u, b^+, b^-, t^+, t^-)
+
+For :math:`n_f=5` light flavors we find:
+
+.. math ::
+  \Sigma_{(5)} &= u^+ + d^+ + s^+ + c^+ + b^+\\
+  V_{(5)} &= u^- + d^- + s^- + c^- + b^-\\
+  T_{0,(5)} &= \frac{3}{2}u^+ + \frac{3}{2}c^+ - d^+ -s^+ - b^+\\
+  V_{0,(5)} &= \frac{3}{2}u^- + \frac{3}{2}c^- - d^- -s^- - b^-\\
+  T_1^u &=u^+ - c^+ \\
+  T_1^d &=d^+ - s^+ \\
+  V_1^u &=u^- - c^- \\
+  V_1^d &=d^- - s^- \\
+  T_2^d &=d^+ + s^+ - 2b^+ \\
+  V_2^d &=d^- + s^- - 2b^- \\
+  \mathcal F \sim  \mathcal F_{intrinsic_unified_evol,5} &= \span(\gamma, g, \Sigma_{(4)}, V_{(4)}, V_{0,(4)}, V_1^d, T_{0,(4)}, T_1^d, V_1^u, T_1^u, T_2^d, V_2^d, t^+, t^-)
+
+For :math:`n_f=6` light flavors the intrinsic QED :math:`\otimes` QCD Evolution Basis coincides with the QED :math:`\otimes` QCD Evolution Basis.
+
+- this basis is *not* normalized with respect to the canonical Flavor Basis
+- the basis transformation from the Flavor Basis is implemented in
+  :meth:`~eko.evolution_operator.flavors.pids_from_intrinsic_evol`
+- the factors 3/2 in the definition of :math:`V_{0,(5)}` and :math:`T_{0,(5)}` are needed in order to have an orthogonal basis for :math:`n_f=5`
```

Comment:
I believe you need a new line before a heading - please check in the browser (I thing it might also be written as a warning when building the doc)

Reply:
where is the heading you're talking about @felixhekhorn?

Reply:
I saw, it's  the line below (GitHub assumes the comments are below the line/s of interest, so it was not showing in the preview)